lib/app.go

```diff
@@ -184,7 +185,7 @@ func (p *awsCWLogsInsightsPlugin) startQuery(startTime, endTime time.Time) (*str
 		StartTime:     aws.Int64(startTime.Unix()),
 		LogGroupNames: aws.StringSlice(p.LogGroupNames),
 		QueryString:   aws.String(p.fullQuery()),
-		Limit:         aws.Int64(1),
+		Limit:         aws.Int64(10),
```

Comment:
With this change, now `--return` will return up to 10 log messages matched by the query.